core/commonMain/src/ArgParser.kt

```diff
@@ -527,7 +527,7 @@ open class ArgParser(
      * Otherwise, prints the usage information and terminates the program execution.
      * @throws IllegalStateException in case of attempt of calling parsing several times.
      */
-    fun parse(args: Array<String>): ArgParserResult = parse(args.asList())
+    fun parse(args: Array<out String>): ArgParserResult = parse(args.asList())
```

Comment:
You're the one who knows it better ;)
But since the library is not stable anyway and is used mostly near the entry points of final projects (and not in the other libraries), I suppose we could proceed with this source-compatible change even if it would not be binary compatible.

Reply:
> You're the one who knows it better ;)

I was hoping someone would check this.

> But since the library is not stable anyway and is used mostly near the entry points of final projects (and not in the other libraries), I suppose we could proceed with this source-compatible change even if it would not be binary compatible.

Ok.